src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -160,6 +161,36 @@ private async Task<ObservedDataMapping> copyObservedData(ObservedDataMapping obs
          };
       }
 
+      private Task<IntroFile[]> copyIntroFiles(dynamic qualificationPlan)
+      {
+         IReadOnlyList<IntroFile> introFiles = GetListFrom<IntroFile>(qualificationPlan.Intro);
+         return Task.WhenAll(introFiles.Select(copyIntroFile));
+      }
+
+      private async Task<IntroFile> copyIntroFile(IntroFile introFile)
+      {
+         Task<string> downloadRemoteIntroductionFile() => downloadRemoteFile(introFile.Path, INTRODUCTION_DOWNLOAD_FOLDER, "Introduction");
+
+         var introductionFileAbsolutePath = absolutePathFrom(_runOptions.ConfigurationFolder, introFile.Path);
+         if (!localFileExists(introductionFileAbsolutePath))
+            introductionFileAbsolutePath = await downloadRemoteIntroductionFile();
+
+         if (!localFileExists(introductionFileAbsolutePath))
+            throw new QualificationRunException(IntroductionFileNotFound(introductionFileAbsolutePath));
+
+         var fileInfo = new FileInfo(introductionFileAbsolutePath);
+
+         DirectoryHelper.CreateDirectory(_runOptions.IntroFolder);
+         var fileName = introFile.Name ?? fileInfo.Name;
+         if (!fileName.EndsWith(MARKDOWN_EXTENSION))

```

Comment:
It could be that this file does not have the markdown extension (for example LICENSE in github does not have it!) 
So I am adding it if it is missing